resources/js/shared/components/airlines/AirlineDropdown.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+
+import { Airline, Flight } from "~/api/api.types";
+
+interface AirlinesProps {
+  airlines: Airline[];
+  flight: Flight | null;
+}
+
+export const AirlineDropdown: React.FC<AirlinesProps> = ({
+  airlines,
+  flight,
+}) => {
+  const [selectedAirline, setSelectedAirline] = useState<number | null>(
```

Comment:
lets typescript infer the type.
```suggestion
  const [selectedAirline, setSelectedAirline] = useState(
```